README.md

```diff
@@ -434,7 +459,8 @@ The NativeScript CLI calls the SDK for the selected target platform and uses it
 
 When you build for Android, the NativeScript CLI saves the application package as an `APK` in `platforms` &#8594; `android` &#8594; `bin`.
 
-When you build for iOS, if the `--device` flag is not set, the NativeScript CLI builds your project for the native emulator and saves the application package as an `APP` in `platforms` &#8594; `ios` &#8594; `build` &#8594; `emulator`. If the `--device` flag is set, the NativeScript CLI builds your project for device and saves the application package as an `IPA` in `platforms` &#8594; `ios` &#8594; `build` &#8594; `device`.
+When you build for iOS, the NativeScript CLI will either build for a device, if there's a device attached, or for the native emulator if there are no devices attached. In order to trigger a native emulator build with an attached device, the `--emulator` flag has to be set.
+The native emulator build is saved as an `APP` in `platforms` &#8594; `ios` &#8594; `build` &#8594; `emulator`. The device build is saved as an `IPA` in `platforms` &#8594; `ios` &#8594; `build` &#8594; `device`.
```

Comment:
If you want this to appear on a new line, place two spaces or a line breaker at the end of the previous paragraph. Otherwise, this will be rendered as part of the previous paragraph.